### DISCUSSION TITLE
Menu/Interface gone!

### DISCUSSION THREAD

**Discussion by sandrosem** [comfyanonymous/ComfyUI#1086] [Q&A]
Hi everybody!

My menu/interface is gone and I can't seem to restore it. A complete reinstall also didn't help. Anyone got an idea how to get it to show up again? I love Comfy UI but it's not usable without it...

**Comment by ltdrdata** [comment]
Did you installed customs nodes?

**Reply by sandrosem** [reply]
I did. But I de-installed everything and re-installed Comfy UI from scratch without any custom node (can't access the menu to do so anyway).

**Reply by ltdrdata** [reply]
Please check if the menu is still missing when you open ComfyUI in secret mode in your browser.

**Reply by sandrosem** [reply]
oh yeah, it's back (sorry, I didn't think of testing that). do you have any idea on how to get it to work in normal mode?

**Reply by ltdrdata** [reply]
It seems like something in the browser cache is causing interference. Try remove cache for 127.0.0.1

**Reply by sandrosem** [reply]
cleared cache and cookies. still not working....

**Reply by Ferniclestix** [reply]
its probably zoom related, try zooming in or out and using the browser zoom settings. the menu has the ability to escape the page bounds

**Reply by sandrosem** [reply]
hey! tested zooming in and out to the extreme. didn't do the trick either, unfortunately. however, I noticed that if I resize the browser window to a very small one, the menu reappears, docked to the edge of the browser window on the right, across the full height. unfortunately, the window is then too small to be useful.

**Reply by aegis72** [reply]
Same here. Weird.

**Reply by Anaiveterrorpassed** [reply]
I don't know if I can clear cache for just 127.0.0.1..
I thought it clear cache for what all the sites that were loaded onto the browser unless I'm missing something.

**Reply by arroyoDev** [reply]
so frustrating

**Reply by jdpilgrim** [reply]
try 
1. resizing the window (so you can see the menu) 
2. put your mouse pointer over the menu
3. zoom out (i.e. using the mouse)

**Reply by oHuflying** [reply]
resizing the window is work.

**Reply by thorpemark** [reply]
YEs.. the silly mousewheel controls that make teh workflow small but does nothing to help find the menu got me for awhile.. until I saw the post about resizing the window

**Comment by barepixels** [comment]
Might be browser's extension then.  Try a different browser?

**Reply by sandrosem** [reply]
don't think it was the browser as it was working fine in the same browser until earlier today. I also haven't installed any extensions. I'm using Brave browser.

**Comment by sandrosem** [comment]
guys! I just accidentally found the solution: starting comfy ui while another instance is already running and then closing both made the menu reappear. 

Thank you everybody for all your tips and suggestions!

**Reply by eumelzocker** [reply]
2 instances on the same port? How this?

**Reply by sandrosem** [reply]
I'm sorry, I meant a second browser window with comfyui running.

**Reply by mcandre** [reply]
Thank you for posting the solution.

I have already triggered this bug multiple times. I wish to lock the zoom permanently, so that it doesn't trigger when I accidentally brush the touchpad.

**Comment by alumcreek** [comment]
If using the Brave Browser on a Mac and you encounter this quirk where the ComfyUI disappears (it keeps happening to me) go to View -> Actual Size and the menu will reappear, then zoom in manually to get back to editing ComfyUI

**Reply by sandrosem** [reply]
I'm on windows :) but anyway, I found a solution (seems to have been an issue with an older version of comfy anyways as the problem hasn't occured ever since).

**Reply by Anaiveterrorpassed** [reply]
I'm having the same issue.  For me it is when ever I install a bunch of nodes it happens and I try to isolate the node(s) by uninstalling them or removing the folders yet I'm gonna try all of these and report back. I use a tone of browser and mainly Edge and Opera GX. I used it more on Edge because Opera Gx has VPN on and sometimes won't run the port even after I turn it off so let's go!!

**Reply by FrancescoSaverioZuppichini** [reply]
you are a life saver

**Comment by aegis72** [comment]
**Hey all, this is what worked for me.** 

I think that the "save menu position" checkbox:
![image](https://github.com/comfyanonymous/ComfyUI/assets/118572301/441613eb-2d02-4ef1-8a41-4831c9fccbd3)
can set the location off screen; in my case, I have an 21:9 UW monitor and a 16:10 monitor. If I put the UI on the larger one, it places the menu on the far right...in an area  that is now offscreen for the 16:10 monitor if I open a window there.

However, if I clear this checkbox, The position isn't saved, and so a page refresh replaces the menu on the right side of whatever the window is. Here's proof:

I opened the UI on the 21:9, and then made a new window on Monitor 16:10, with "save menu position" checked:
![image](https://github.com/comfyanonymous/ComfyUI/assets/118572301/62f308c3-1186-4c55-b190-6f4780870875)
See that the menu is missing. But if I uncheck the box:

16:10-
![image](https://github.com/comfyanonymous/ComfyUI/assets/118572301/349600d2-9085-4454-9b57-53f20d7c478c)

and 21:9:
![image](https://github.com/comfyanonymous/ComfyUI/assets/118572301/276c8cc0-2aea-49b2-90ff-c7f6af8ed45b)

The issue seems pretty clear cut, and this is a reproducible behavior and workaround :). The problem isn't custom nodes, etc----I install nearly every node available---if anyone would have this issue due to nodes, it would be me.

**Comment by registereduser** [comment]
Another solution:
As suggested above, shrink the window width and the Menu should appear but rather big, with the mouse over the menu, zoom out, it will then scale back down and move to the correct location.
